README.md

```diff
@@ -1,139 +1,466 @@
+    Stylelint warning is triggered.
+
+2.  To ignore several lines, add `// stylelint-disable` before the code in
+    question and add `// stylelint-enable` afterwards.
+
+The Stylelint rules can be changed in the `.stylelintrc.yml` file. By default,
+Gesso follows the
+[sass-guideline.es](https://github.com/bjankord/stylelint-config-sass-guidelines)
+and [Prettier’s recommended
+guidelines](https://github.com/prettier/stylelint-config-prettier), with some
+additional customizations.
+
+The Prettier config can be changed in the `.prettierrc` file.
 
 ## JavaScript
-JavaScript can be compiled to individual JS files for use in a JavaScript library or included
-within a different JS file. JS files that use modern (ES2015+) syntax must be named
-`[name].es6.js`, but this is not required by the compiler. JavaScript files should go in the
-appropriate folder under source (e.g. `source/03-components/menu` for menu-related JavaScript).
-There is not a separate folder for JS files as there was in previous versions of Gesso.
 
-All JavaScript files must have a unique name, even if they are in different directories.
+JavaScript can be compiled to individual JS files for use in a JavaScript
+library or included within a different JS file. JS files that use modern
+(ES2015+) syntax must be named `[name].es6.js`, but this is not required by the
+compiler. JavaScript files should go in the appropriate folder under source
+(e.g., `source/03-components/menu` for menu-related JavaScript). There is not a
+separate folder for JS files as there was in previous versions of Gesso.
+
+All JavaScript files must have a unique filename, even if they are in different
+directories.
 
 ### Modules
-Prefix the name of your JavaScript file with `_`, e.g. `_Menu.es6.js`. Import it to the appropriate
-JavaScript file(s), (i.e. `primary-menu.es6.js`).
 
-### Individual Component/Library Scripts
+Prefix the name of your JavaScript file with `_`, e.g. `_Menu.es6.js`. Import it
+to the appropriate JavaScript file(s), (i.e. `primary-menu.es6.js`).
+
+### Individual component/library scripts
+
 DO NOT prefix the name of your JS file with `_`. Import your JS file at the top
-of your Storybook file. See `menu.stories.jsx` for an example. Don't forget to
+of your Storybook file. See `menu.stories.jsx` for an example. Don’t forget to
 add it to the `gesso.libraries.yml` file as well.
 
 ### Linting
-ESLint and Prettier are used to lint JavaScript files. ESLint uses the [Airbnb
-standards](https://github.com/airbnb/javascript/), which are [followed by Drupal](https://www.drupal.org/docs/develop/standards/javascript/javascript-coding-standards) as well.
+
+ESLint and Prettier are used to lint JavaScript files. If you have a valid
+reason to break one of the rules, you can ignore a specific line using any of
+the options in the [ESLint
+documentation](https://eslint.org/docs/user-guide/configuring#disabling-rules-with-inline-comments).
```

Comment:
If you do take the suggestion to encourage devs to write a comment with the reason, it's probably worth repeating here.

Reply:
Done